splunk/centos-7/Dockerfile

```diff
@@ -11,61 +11,64 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM base-centos-7:latest
-LABEL maintainer="support@splunk.com"
 
+#
+# Download and unpack Splunk Enterprise
+#
+FROM base-debian-9:latest as package
```

Comment:
Should be base-centos-7 here (if we want to continue including centos as part of this repo)

Reply:
Keep forgetting this is multi-stage - I guess it shouldn't matter if this is the just an ephemeral stage

Reply:
Yeah, I don't think it matters in this case.. just left it to be consistent across platforms.